GameDesign/Assets/Scripts/Events/GameEventTrigger.cs

```diff
@@ -31,6 +33,12 @@ public void TriggerEventViaCode(Collider other)
 
         if (EventTriggerCondition(other))
         {
+            Debug.LogError(PlayerPrefs.GetInt(m_event.name, 0));
```

Comment:
This shouldnt be a LogError, it stops the game when you get to this point, if this code is running